client/pages/sections/protocols/sections.js

```diff
@@ -106,13 +110,68 @@ const getBadges = (section, newComments, values) => {
 
   const fields = getFieldKeys(section, values);
 
+  // Initialize groups for fields with and without values
+  const fieldsWithValues = [];
+  const fieldsWithoutValues = [];
+
+  section.fields?.forEach((field, index) => {
+    console.log("Processing field:", field.name);
+
+    // Attempt to retrieve the value from the values object
+    const rawValue = field.name.includes('.')
+      ? field.name.split('.').reduce((acc, key) => acc?.[key], values)
+      : values?.[field.name];
+
+    let fieldValue;
+
+    if (typeof rawValue === 'object' && rawValue !== null) {
+      // Handle structured data
+      if (rawValue.object) {
+        fieldValue = rawValue.object; // Extract `object` key if available
+      } else if (Array.isArray(rawValue)) {
+        fieldValue = rawValue.join(', '); // Join array elements
+      } else if (rawValue.document) {
+        fieldValue = JSON.stringify(rawValue.document); // Handle `document` key
+      } else {
+        fieldValue = JSON.stringify(rawValue); // Convert object to string
+      }
+    } else {
+      // Handle simple values or fallback to default message
+      fieldValue = rawValue || null;
+    }
+
+    console.log(`Field ${index + 1}: ${field.name} = ${fieldValue}`);
```

Comment:
```suggestion
```

Reply:
These logs are going to be very noisy in opensearch when doing server-side rendering of the page, and could potentially be logging PII which would be a security risk